client/shade/pom.xml

```diff
@@ -279,6 +279,13 @@
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                                     <manifestEntries>
                                         <Multi-Release>true</Multi-Release>
+                                        <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
+                                        <Bundle-SymbolicName>org.jboss.client</Bundle-SymbolicName>
+                                        <Bundle-Version>1.0</Bundle-Version>
+                                        <Bundle-Name>Jakarta Enterprise Beans and Jakarta Messaging client library</Bundle-Name>
+                                        <ExtensionFragment-Host>org.openjdk.jmc.rjmx</ExtensionFragment-Host>
```

Comment:
Ups, sorry @moulalis for all the back and forth on the review, could you please replace `ExtensionFragment-Host` by `Fragment-Host`, I've just realized `ExtensionFragment-Host` is not a valid OSGI bundle property. It seems the Jira description itself is badly formatted and hence all the confusion.

Reply:
It's done, Thanks.

Reply:
thank-you!